section1/reflection.md

```diff
@@ -2,18 +2,36 @@
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+I would use a \\ to leave a code comment in a js file. Code comments are valuable to explain and reason to your team about changes you may have made to the project. If you deactivated certain portions of code, for example, you could explain in your own words why you did that or how you decided to do it differently.
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+A variable is a place where we store values to use and reuse, like a jar in a pantry. We store something inside of that var-jar in order to easily access some object/information that we put in it. That way, when we have to reference that piece of information often, we have instant access to it; it's organized. We can close the jar but we can always open it again to change the contents.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+One app that I use is Gmail. There are certain phrases that Gmail repeats, like 'moved to trash' or 'message sent'. I imagine these strings are stored as variable, that happen as a result of pushing a button, etc. I also could see how the # of messages in your inbox is probably calculated by a set variable which adds together the number of all unread emails to display at the top of the inbox. Gmail also probably stores yourName as a variable, to input when it talks to you specifically.  
+
 7. In your own words, explain what concatenation is.
 
+Concatenation is how we join together elements of code that may be different from one another. That can be joining any number of variables as numbers, names or strings. The tool does so by using type coercion so that several different values can be represented in one place.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+An app I use frequently is Reddit. The developers maybe used concatenation to effectively format their upvote/downvote system. Because the upvotes and downvotes both influence the overall vote, they would have to create something that combined the output of the two to create the overall # of votes.
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+Questions 6 and 7 were a little difficult for me to answer. It felt like I was reaching to try to imagine how these different elements work together to make a website or app run efficiently. I probably just need more practice but some examples from instructors would be nice! I also wonder if programmers are working on a project together would they have to streamline the variables they were using? Otherwise wouldn't they be creating multiple variables for the same thing which could mess with the outcome? Also, what exactly is the modulo?
```

Comment:
Your answers for 6 and 7 are spot on, I love the jar analogy.

In Slack, there is probably a variable for the workspace "Turing" the username, and each part of the profile "pronouns" "description" "email" etc.

Nice job on Section 1 and throughout this reflection!